miden/tests/integration/flow_control/mod.rs

```diff
@@ -352,3 +353,55 @@ fn simple_dyncall() {
         false,
     );
 }
+
+// PROCREF INSTRUCTION
+// ================================================================================================
+
+#[test]
+fn fmpadd() {
+    let module_source = "
+    export.foo
+        push.1.2
+    end";
+    let module_ast = ModuleAst::parse(module_source).unwrap();
+    let library_path = LibraryPath::new("module::path::one").unwrap();
+    let module = Module::new(library_path, module_ast);
+    let masl_lib = MaslLibrary::new(
+        LibraryNamespace::new("module").unwrap(),
+        assembly::Version::default(),
+        false,
+        vec![module],
+        vec![],
+    )
+    .unwrap();
+
+    let source = "
+    use.module::path::one
```

Comment:
Instead of building a custom library, could we not use some procedure from `stdlib` - maybe something from `math::u64` module?